src/Styles/Header.scss

```diff
@@ -0,0 +1,33 @@
+@import 'Mixins';
+
+
+// headingText co">CO</div>
+//         <div className="headingText yo">YO</div>
+//         <div className="headingText coyoHeading">Co
+
+
+.banner {
+  width: 100%;
+  background: no-repeat bottom url("./colorado-mountains.jpg");
+  filter: brightness(80%);
+  color: #eff0f2;
+  text-align: center;
+  padding: 100px;
+}
+
+.co,
```

Comment:
Is there a better class name we could us here?

Reply:
I've changed these to .coHeadingText and .yoHeadingText.